gfw_pixetl/layer.py

```diff
@@ -474,36 +474,45 @@ def translate(self, tiles: Iterator[RasterSrcTile]) -> Iterator[RasterSrcTile]:
 
             cmd: List[str] = (
                 [
-                    "gdal_translate",
-                    "-strict",
-                    "-a_nodata",
-                    "none",  # ! important
+                    "gdalwarp",
```

Comment:
gdalwrap allows to do the same thing as gdal_translate plus transforming from one projection to another. Using this  tool allows us to use any kind of input src and reproject it to WGS84